components/ai_chat/content/browser/page_content_fetcher.cc

```diff
@@ -404,18 +505,23 @@ void FetchPageContent(content::WebContents* web_contents,
     }
   }
 #endif
+  auto* fetcher = new PageContentFetcher();
```

Comment:
should fetcher/loader only be created if patch_url is nonempty?

Reply:
It should be created even if it is empty because it will be used in the below block if not there:
>   mojo::Remote<mojom::PageContentExtractor> extractor;
> // GetRemoteInterfaces() cannot be null if the render frame is created.
> primary_rfh->GetRemoteInterfaces()->GetInterface(
      extractor.BindNewPipeAndPassReceiver());
> fetcher->Start(std::move(extractor), invalidation_token, loader,
                 std::move(callback));